src/test/java/og_spipes/rest/ScriptControllerTest.java

```diff
@@ -57,7 +58,9 @@ public void init() throws Exception {
         }
         FileUtils.copyDirectory(new File("src/test/resources/scripts_test/sample/hello-world"), scriptsHomeTmp);
         FileUtils.copyFileToDirectory(new File("src/test/resources/SHACL/rule-test-cases/data-without-label.ttl"), scriptsHomeTmp);
-        mapper.registerModule(new JsonLdModule());
+        JsonLdModule module = new JsonLdModule();
```

Comment:
those 3 lines look like should be wrapped into a single method within some kind of "test util" class, what do you think?